linear_api/managers/issue_manager.py

```
@@ -738,14 +742,125 @@ def get_comments(self, issue_id: str) -> List[Comment]:
+            "filename": filename,
+            "size": len(file_data)
+        }
+
+        response = self._execute_query(mutation, variables)
+
+        if (
+            not response
+            or "fileUpload" not in response
+            or not response["fileUpload"].get("uploadFile")
+        ):
+            raise ValueError("Failed to get file upload URL")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)